samples/bot-all-cards/nodejs/.notification.testtoolstore.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
.notification.testtoolstore file needed or not?

Reply:
yes this file is needed